scripts/test/runtests.ps1

```diff
@@ -5,6 +5,14 @@
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
+$failCount = 0
+
+# Capture PATH for later
```

Comment:
That was just an incomplete removal of code from @brthor 's earlier comment :). Good catch